pkg/kubectl/cmd/get.go

```diff
@@ -266,17 +267,22 @@ func RunGet(f *cmdutil.Factory, out io.Writer, errOut io.Writer, cmd *cobra.Comm
 
 		first := true
 		filteredResourceCount = 0
-		kubectl.WatchLoop(w, func(e watch.Event) error {
-			if !isList && first {
-				// drop the initial watch event in the single resource case
-				first = false
-				return nil
-			}
-			err := printer.PrintObj(e.Object, out)
-			if err == nil {
+		intr := interrupt.New(nil, w.Stop)
+		intr.Run(func() error {
```

Comment:
Do you want to add a 2nd commit to return the result of this call instead of nil, as you suggested in the previous PR?